src/apps/reservation-list/reservation-material/reservation-info.tsx

```diff
@@ -81,10 +81,11 @@ const ReservationInfo: FC<ReservationInfoProps> = ({
         color={success as string}
         percent={100}
         info={getInfo()}
-        label={[
-          pickupLibrary,
-          pickupNumber || t("reservationListDigitalPickupText")
-        ]}
+        label={
+          !isDigital
+            ? [pickupLibrary, pickupNumber || ""]
+            : [t("reservationListDigitalPickupText")]
+        }
```

Comment:
I guess my speculations [here](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/664/files#r1387828611) turned out to be true 😅